src/abstractitensornetwork.jl

```diff
@@ -646,6 +654,23 @@ function flatten_networks(
   return flatten_networks(flatten_networks(tn1, tn2; kwargs...), tn3, tn_tail...; kwargs...)
 end
 
+#Ideally this will dispatch to inner_network but this is a temporary fast version for now
+function unflattened_norm_network(tn::AbstractITensorNetwork)
```

Comment:
Maybe just `norm_network`? We could add some options to choose if it outputs flattened or unflattened in the future.